lib/trento/infrastructure/commanded/middleware/enrich_register_cluster_host.ex

```diff
@@ -22,6 +34,36 @@ defimpl Trento.Infrastructure.Commanded.Middleware.Enrichable,
         error
     end
 
-    {:ok, command}
+    {:ok,
+     %RegisterClusterHost{
+       command
+       | details: strip_irrelevant_details(type, sid, details)
+     }}
+  end
+
+  defp strip_irrelevant_details(
+         ClusterType.hana_scale_up(),
+         sid,
+         %HanaClusterDetails{nodes: cluster_nodes} = details
+       ) do
+    %HanaClusterDetails{
+      details
+      | nodes: Enum.map(cluster_nodes, &strip_irrelevant_node_data(&1, sid))
+    }
+  end
+
+  defp strip_irrelevant_details(_, _, details), do: details
+
+  defp strip_irrelevant_node_data(
+         %HanaClusterNode{attributes: attributes} = node,
+         sid
+       ) do
+    {_, new_attributes} =
+      Map.split_with(attributes, fn {key, _value} -> key == "lpa_#{String.downcase(sid)}_lpt" end)
```

Comment:
Now that we don't want this get the modified version, maybe simply `new_attributes = Map.drop(attributes, ["lpa_#{String.downcase(sid)}_lpt")`